DevTools/Airlock/explainer.md

```diff
@@ -0,0 +1,61 @@
+# DevTools: Airlock, a new tool for DOM node memory leaks
+
+Authors:
+ - [Brian Cui](https://github.com/brcui), Software Engineer 2
+ - [Guangyue Xu](https://github.com/guangyuexu), Software Engineer
+ - [Jose Leal Chapa](https://github.com/joselea), Senior Software Engineer
+ - Josephine Le, Designer
+ - [Sofian Hnaide](https://github.com/SofianHn), Principal Software Architect
+ - [Zoher Ghadyali](https://github.com/zoherghadyali), Program Manager
+
+## Status of this feature
+This feature has not yet shipped in Microsoft Edge and is in active development. See the [Changelog section](#changelog) for details on progressive feature implementation.
+
+The team welcomes your feedback on this feature; please [file an issue](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/new?labels=DevTools&template=devtools.md&title=%5BDevTools%5D+%3CTITLE+HERE%3E) to share your thoughts.
+
+## Introduction
+**Airlock** is a new tool, originally built by the Microsoft Teams team as a DevTools extension to help them find and debug memory leaks. However, given the limited API surface for DevTools extensions, Airlock isn't able to use the [Chromium DevTools Protocol (CDP)](https://chromedevtools.github.io/devtools-protocol/) itself, which forced users of the extension to rely on the **Memory** tool in the Microsoft Edge DevTools to trigger garbage collection (GC) and take heap snapshots. By integrating Airlock as its own tool in the Microsoft Edge DevTools, we can provide innovative, new tooling for debugging DOM node memory leaks.
+
+## Goals
+
+  1. Provide Airlock as a new tool in the Microsoft Edge DevTools to better debug DOM node memory leaks
+
+We're eager to hear your feedback about Airlock and the current state of memory tooling in the Microsoft Edge DevTools. Let us know what you think by filing an [issue on this repo](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/new?labels=DevTools&template=devtools.md&title=%5BDevTools%5D+%3CTITLE+HERE%3E) or on [Twitter](https://twitter.com/EdgeDevTools).
+
+## Usage
+
+**Note:** Airlock is not currently available in the Microsoft Edge DevTools. The [Changelog section](#changelog) will be updated as soon as a version of Microsoft Edge with Airlock is available.
+
+### What is a DOM node memory leak?
+
+A DOM node memory leak is when a DOM node that is removed from the DOM tree cannot be garbage collected because it is still referenced by the page's JavaScript. Nodes that fall into this category are considered "detached". Depending on the parents and children of the node that causes the leak, a DOM node memory leak can span from just a few nodes all the way to massive trees of hundreds or thousands of elements. This can also be a common occurrence for React apps because React maintains a virtual representation of the DOM in JavaScript.
```

Comment:
```suggestion
A DOM node memory leak occurs when a DOM node that is detached from the DOM tree cannot be garbage collected because a reference to it still exists. Nodes that fall into this category are considered "detached". A detached node's parent and child references can cause a single node memory leak to span just a few nodes all the way to massive trees of hundreds or thousands of elements. This can also be a common occurrence for React apps because React maintains a virtual representation of the DOM in JavaScript.
```